main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
+// All the questions that the program asks the user
+    noun = input("Choose a noun: ")
+    plural_noun = input("Choose a plural noun: ")
+    second_noun = input("Choose a noun: ")
+    place = input("Name a place: ")
+    adjective = input("Choose an adjective (Describing word): ")
+    third_noun = input("Choose a noun: ")
```

Comment:
Consider adding the user input to a data structure, like a dictionary where the keys are the questions and the values are the variable names, so that the user input can be easily accessed to display the story

`
dict = {"Choose a noun: “: None, "Choose a plural noun: ": None} #and so on...
`
`
for key in dict:
`
`
    dict[key] = input(value)
`